logs_ingest/main.py

```diff
@@ -102,7 +102,7 @@ def is_too_old(timestamp: str, self_monitoring: SelfMonitoring, log_part: str):
                 return True
         except Exception:
             # Not much we can do when we can't parse the timestamp
-            logging.debug(f"Failed to parse timestamp {timestamp}")
+            logging.exception(f"Failed to parse timestamp {timestamp}", "timestamp-parsing-debug")
```

Comment:
[Minor] please change timestamp-parsing-debug to timestamp-parsing-exception

Reply:
Fixed